src/vos/tests/evt_ctl.c

```diff
@@ -1682,6 +1683,239 @@ test_evt_various_data_size_internal(void **state)
 	}
 }
 
+static void
+test_evt_node_size_internal(void **state)
+{
+	struct test_arg		*arg = *state;
+	daos_handle_t		 toh;
+	int			order_size, rc;
+	bool			evt_created;
+
+	for (order_size = 0; order_size < ORDER_TEST_SIZE; order_size++) {
+		evt_created = false;
+		rc = evt_create(arg->ta_root, ts_feats, order_size,
+				arg->ta_uma, &ts_evt_desc_cbs, &toh);
+		if ((order_size >= EVT_MIN_ORDER) &&
+		(order_size <= EVT_MAX_ORDER)) {
+			assert_int_equal(rc, 0);
+			evt_created = true;
+		} else {
+			assert_int_not_equal(rc, 0);
```

Comment:
Do we know what the expected error code is if the order is outside min/max?

Reply:
We get DER_INVAL as error message. I think this is possible to get same error message for evt_create (for other input parameters). I wouldn't rely on those values. There is no specific error message for order is outside min/max.